docs/settingUp.md

```diff
@@ -32,12 +40,18 @@ More information can be found at [this documentation](https://help.github.com/ar
    ```sh
    ./gradlew appengineDownloadSdk
    ```
+   
+ **Verification** : Check the gradle folder for the existence of the SDK.The folder path will be like this : 
```

Comment:
@wkurniawan07 I think its better if we add the version number also right ?.So that dev can confirm the version also ?